File: Majo/Act-Instructor_Henry/lenguajes/ruby/junio/19-06-24.rb
puts 'Cantidad de aprendices:'
n = gets.chomp.to_i

(1..n).each do
    a = ''
    b = ''
    c = ''

        puts 'Ingrese un nombre: '
        nombre = gets.chomp
        if nombre.length >= 2
        a = nombre[-2, 2]
        end

        puts 'Ingresa un municipio: '
        municipio = gets.chomp
        if municipio.length >= 3
        b = municipio[0, 3]
        end

        puts 'Ingrese la placa: '
        placa = gets.chomp
        if placa.length >= 3
        c = placa[-3, 3]
        end

    puts "Su código es: #{b}#{a}#{c}"
end

File: Majo/Act-Instructor_Henry/lenguajes/dart/junio/19-06-24.dart
import 'dart:io';

void main() {
    print('Cantidad de aprendices:');
    int n = int.parse(stdin.readLineSync()!);

  for (var x = 1; x <= n; x++){
  
    String a = '';
    String b = '';
    String c = '';
    print('Ingrese un nombre: ');
    String? nombre = stdin.readLineSync();
    if (nombre != null && nombre.length >= 2) {
      a = nombre.substring(nombre.length - 2);
    } 
    
    print('Ingresa un muncipio: ');
    String? municipio = stdin.readLineSync();
    if (municipio != null && municipio.length >= 3) {
      b = municipio.substring(0 , 3);
      
    }
    
    print('Ingrese la placa: ');
    String? placa = stdin.readLineSync();
    if (placa != null && placa.length >= 3){
      c = placa.substring(placa.length - 3);
      
    }
    

    print('Su codigo es: $b$a$c ');
  }
}
